src/IscsiLioData.ycp

```diff
@@ -225,6 +225,18 @@ boolean IsTpgActive( string tgt, integer tpg )
     return( data["tgt",tgt,tpg,"ep","enabled"]:false );
     }
 
+boolean LogExecAuthCmd( string cmd, string user, string password )
```

Comment:
name is quite confusing as primary goal is execute...I think simple ExecAuthCmd is enough, logging is just side effect.